neo/Prompt/Commands/tests/test_loadsmartcontract.py

```diff
@@ -0,0 +1,99 @@
+from unittest import TestCase
+from neo.Prompt.Commands.LoadSmartContract import LoadContract, GatherLoadedContractParams
+import mock
+
+
+class LoadSmartContractTestCase(TestCase):
+
+    def test_loadcontract(self):
+
+        # test too few args
+        args = []
+
+        res = LoadContract(args)
+
+        self.assertFalse(res)
+
+        # test for void (ff) type in params
+        with self.assertRaises(Exception) as e:
+            args = ["path", "07ff10", "01", "False", "False", "False"]
+
+            LoadContract(args)
+
+        self.assertTrue("Void is an unsupported input type" in str(e.exception))
+
+        # test for .py in path
+        args = ["path.py", "070710", "01", "False", "False", "False"]
+
+        res = LoadContract(args)
+
+        self.assertFalse(res)
+
+        # test good contract
+        with mock.patch("builtins.open", mock.mock_open(read_data="path.avm")):
+
+            args = ["path.avm", "070710", "01", "True", "True", "True"]
+
+            res = LoadContract(args)
+
+        self.assertTrue(res)
+
+        # test bad contract

```

Comment:
I believe we're testing for a "file not found" right? If so, can you please update the comment to clarify that. At first I was looking at the params wondering what's bad about the contract or its usage. Just a minor thing, but will help if we ever need to review it again.